src/coreclr/src/gc/gcpriv.h

```diff
@@ -55,6 +55,13 @@ inline void FATAL_GC_ERROR()
 // turned on.
 #define FEATURE_LOH_COMPACTION
 
+// ARM requires that 64-bit primitive types are aligned at 64-bit boundaries for interlocked-like operations.
+// Additionally the platform ABI requires these types and composite type containing them to be similarly
+// aligned when passed as arguments.
+#ifdef TARGET_ARM
+#define FEATURE_64BIT_ALIGNMENT
+#endif
```

Comment:
this is already defined in switches.h so the embedded GC already does this. could we include this in a standalone GC header like gcenv.object.h?

Reply:
It may be better to define it in gcenv.object.h. I would not include switches.h into standalone GC. 95% of switches.h has nothing to do with GC.

Reply:
yep that's what I meant :) to include the lines above in gcenv.object.h.

Reply:
I've just pushed a commit to do this.